client/src/app/+admin/users/user-edit/user-update.component.ts

```diff
@@ -19,9 +19,12 @@ import { UserService } from '@app/shared'
 export class UserUpdateComponent extends UserEdit implements OnInit, OnDestroy {
   error: string
   userId: number
+  userEmail: string
   username: string
+  isAdministration = false
```

Comment:
I don't understand why we need this, because we are in the `+admin/` directory so `isAdministration` should always be `true`